kernelci/data/kernelci_api.py

```diff
@@ -72,6 +72,14 @@ def get_node(self, node_id):
         resp = self._get('/'.join(['node', node_id]))
         return json.loads(resp.text)
 
+    def get_node_by_commit_id(self, commit_id):
+        """
+        Get node object by sending commit ID to 'nodes' endpoint
```

Comment:
It's actually the git commit SHA1, using "id" might be a bit ambiguous here as it might be confused with a database id.

Reply:
Renamed the function.